.github/workflows/security-scan.yml

```diff
@@ -0,0 +1,356 @@
+---
+name: "Security Scanning"
+
+on:
+  push:
+    branches:
+      - "main"
+      - "release-*"
+  pull_request:
+    branches:
+      - "main"
+      - "release-*"
+  schedule:
+    - cron: '0 12 * * 1'
+
```

Comment:
Wouldn't it be good to also make possible to explicitly invoke this workflow from the GitHub web UI? Could be done in a follow-on PR.